index.html

```diff
@@ -47,6 +47,148 @@ <h2 class="secondary-text">I'am a Software Developer</h2>
     </div>
   </section>
   </div>
-</body>
 
+  <div class="works">
+    <div class="title">
+      <a id="recent" >My Recent Works</a>
```

Comment:
This should use a `<hx>` tag for semantic  correctness and to enable screen readers read your page.